frontend/components/SwipeListElement/index.js

```diff
@@ -0,0 +1,109 @@
+import React, { useRef } from 'react';
+import { Animated, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import { Swipeable, RectButton } from 'react-native-gesture-handler';
+import { Button, withTheme } from 'react-native-paper';
+import ListElement from '../ListElement';
+// const AnimatedView = Animated.createAnimatedComponent(View);
+
+// const renderRightActions = (progress, dragX) => {
+// 	const scale = dragX.interpolate({
+// 		inputRange: [-80, 0],
+// 		outputRange: [1, 0],
+// 		extrapolate: 'clamp',
+// 	});
+// 	return (
+// 		<RectButton style={styles.rightAction}>
+// 			<AnimatedView style={[styles.actionItems, { transform: [{ scale }] }]} />
+// 		</RectButton>
+// 	);
+// };
```

Comment:
Do we still need this for future reference? If we do, leave a line explaining why. If not, let's delete it.